cmd/juju/commands/debuglog.go

```diff
@@ -163,16 +163,16 @@ func (c *debugLogCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.Var(cmd.NewAppendStringsValue(&c.params.IncludeLabel), "include-label", "Only show log messages for these logging labels")
 	f.Var(cmd.NewAppendStringsValue(&c.params.ExcludeLabel), "exclude-label", "Do not show log messages for these logging labels")
 
-	f.StringVar(&c.level, "l", "", "Log level to show, one of [TRACE, DEBUG, INFO, WARNING, ERROR]")
+	f.StringVar(&c.level, "l", "", "Log level to show, one of [TRACE, DEBUG, INFO, WARNING, ERROR]. Logs of higher severity (e.g. ERROR) will still be shown when a lower level (e.g. INFO) is selected")
```

Comment:
We don't need this sort of explanation here. The arg description is supposed to be short and succinct. Any clarifications like this can go into the body of the main help text.

Reply:
Sure let me refactor this, thanks Ian :)